inference/README.md

```diff
@@ -26,31 +29,37 @@ docker compose --profile inference up -d
 Tail the logs:
 
 ```shell
+# cd /root/of/this/repository
 docker compose logs -f    \
     inference-server      \
     inference-worker      \
-    inference-text-client \
-    inference-text-generation-server
+    inference-safety
```

Comment:
inference-safety is removed from inference profile in recent main

Reply:
OK.  I will kill it here too.